pkg/install/deploystorage.go

```diff
@@ -226,9 +245,7 @@ func (i *Installer) deployStorageTemplate(ctx context.Context, installConfig *in
 								"*/delete",
 								"*/write",
 							},
-							NotActions: &[]string{
-								"Microsoft.Network/networkSecurityGroups/join/action",
-							},
+							NotActions: notActions,
```

Comment:
I am now looking at how to scope these exceptions to the deny assignment more narrowly than `ResourceGroupID` so that they only apply to Velero.

Reply:
not sure I understand.  I expect the scope of this feature to be all backup solutions, not just Velero.
I don't think there's a meaningful way to scope the notActions more than they are.

Reply:
Agree - Velero is just the POC case but we shouldn't block e.g. customers writing their own homegrown backups system if they want.